discord-scripts/fjord.ts

```diff
@@ -254,54 +258,40 @@ export default async function manageFjord(
           axios
             .get(`${webhookUrl}?${queryParams.toString()}`, options)
             .then(async (response) => {
-              if (message.channel.type === ChannelType.GuildPublicThread) {
-                channel.send(response.data)
-              } else {
-                const threadName = `${repositoryOwner}/${repositoryName} - git activity`
-                await handleThreadCreate(
-                  message,
-                  threadName,
-                  "git activity",
-                  response.data,
-                )
+              if (channel instanceof TextChannel) {
+                const thread = await channel.threads.create({
+                  name: `Git Activity: ${repositoryOwner} ${repositoryName}`,
+                  autoArchiveDuration: 60,
+                })
+                await thread.send("@here")
+                await thread.send(response.data)
               }
             })
             .catch((error) => {
-              channel.send(
-                `Automation git activity flow failed: ${error.message}`,
+              interaction.followUp(
+                `Automation activity flow failed: ${error.message}`,
               )
             })
-        } else {
-          channel.send(
-            "**Please provide these parameters: !activity <username> <repo> **",
-          )
         }
       }
 
-      if (message.content.startsWith("!exec")) {
-        const parameters = message.content.split(" ").slice(1)
-        if (parameters.length >= 1) {
-          const [workflowName] = parameters
+      if (interaction.isCommand() && interaction.commandName === "exec") {
```

Comment:
Exec is a mega-techy way of saying this. Could we just make this command `/automation` or `/n8n`?